pkg/testutils/bazel.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package testutils
+
+import (
+	"os"
+	"path"
+	"strings"
+	"testing"
+)
+
+const testSrcDirEnv = "TEST_SRCDIR"
+const testTmpDirEnv = "TEST_TMPDIR"
+const testWorkspaceEnv = "TEST_WORKSPACE"
+const testTargetEnv = "TEST_TARGET"
+
+// RunningUnderBazel returns true if the test is executed by bazel.
+func RunningUnderBazel() bool {
+	return os.Getenv(testSrcDirEnv) != ""
+}
+
+func requireEnv(env string) string {
+	if v := os.Getenv(env); v != "" {
+		return v
+	}
+	panic("expected value for env " + env)
```

Comment:
We generally avoid direct string concatenation (citations needed), in favor of something like 

```go
panic(errors.AssertionFailedf("missing env var: %s", env))
````